lighthouse-core/audits/lcp-lazy-loaded.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit.js');
+const i18n = require('../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Descriptive title of a diagnostic audit that provides */
+  title: 'LCP element was lazy-loaded',
+  /** Description of a Lighthouse audit that tells */
+  description: '',
+};
+
+const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
+
+class LargestContentfulPaintElement extends Audit {
+  /**
+   * @return {LH.Audit.Meta}
+   */
+  static get meta() {
+    return {
+      id: 'lcp-lazy-loaded',
+      title: str_(UIStrings.title),
+      description: str_(UIStrings.description),
+      scoreDisplayMode: Audit.SCORING_MODES.ERROR,
+      supportedModes: ['navigation'],
+      requiredArtifacts: ['traces', 'TraceElements', 'ImageElements'],
```

Comment:
I don't think we actually use `traces` anywhere
```suggestion
      requiredArtifacts: ['TraceElements', 'ImageElements'],
```